docs/call-tree.md

```diff
@@ -0,0 +1,353 @@
+# Call Tree
+
+A call tree is a summary representation of sample and stack data. It adds together all of the timings of the different functions, and presents them in a tree depending on how they are called. For instance take the following 3 samples, where the letter represents the function that was being called.
+
+```
+ 1ms  2ms  3ms
+  A    A    A
+  |    |    |
+  v    v    v
+  B    B    B
+  |    |    |
+  v    v    v
+  C    C    H
+  |    |    |
+  v    v    v
+  D    F    I
+  |    |
+  v    v
+  E    G
+```
+
+In this case the profiler took 3 samples, each 1 millisecond apart. This approach doesn't scale when you have thousands of samples, so it's best to summarize this information. The call tree does this by combining areas where functions are shared at the same level of a call stack. Taking the above samples and the corresponding relationship between the stacks would produce the following tree.
```

Comment:
> This approach doesn't scale

Maybe "Visualizing profile information in this form doesn't scale"? Or were you referring to the in-memory data representation? Or maybe both?